pkg/dhcp6client/dhcp6client.go

```diff
@@ -92,7 +96,7 @@ func (p *buffer) remaining() []byte {
 
 func (c *Client) ReadReply() (*dhcp6.Packet, error) {
 	var err error
-	for i := 0; i < 5; i++ { // five attempts
+	for i := 0; i < c.attempts; i++ { // five attempts
 		pb := make([]byte, 1500)
```

Comment:
Is there some constant somewhere that defines the max length? this 1500 is very strange here.